app/assets/stylesheets/scaffold.css

```diff
@@ -0,0 +1,84 @@
+body {
```

Comment:
You should probably write some sass instead and put it in `app/gulp/sass`; it will go through our gulp pipeline that way and also be autoprefixed.

(CSS is valid SCSS, so you can just move the file and change the extension to `scss` if you're really lazy.)